serial_node:
  ros__parameters:
    port_name: "/dev/ttyS1"
    baud_rate: 38400

